modules/adminlog.cpp

```diff
@@ -27,6 +27,7 @@ class CAdminLogMod : public CModule {
 		AddHelpCommand();
 		AddCommand("Show", static_cast<CModCommand::ModCmdFunc>(&CAdminLogMod::OnShowCommand), "", "Show the logging target");
 		AddCommand("Target", static_cast<CModCommand::ModCmdFunc>(&CAdminLogMod::OnTargetCommand), "<file|syslog|both>", "Set the logging target");
+		AddCommand("Path", static_cast<CModCommand::ModCmdFunc>(&CAdminLogMod::OnPathCommand), "<path>", "If using the file target, set the log file path");
```

Comment:
just to throw an idea on the table... perhaps the path could be passed as an optional argument to the target command? not sure which is better.